WAKU.md

```diff
@@ -0,0 +1,16 @@
+# Waku spec support
+
+*Last updated February 14, 2020*
+
+status-go client of Waku is spec compliant with [Waku spec v0.3](https://specs.vac.dev/waku.html) with the exception of:
+- Currently nodes with higher version don't automatically disconnect if versions are different
+
+It doesn't yet implement the following recommended features:
+- It doesn't apply a timeout to receive Status message
+- Partial support for rate limiting
+- No support for DNS discovery to find Waku nodes
+- No support for negotiation with peer supporting multiple version
+- Exchange topic-interest packet code isn't implemented yet (WIP status-im/status-go#1853)
```

Comment:
```suggestion
```

Support this is accurate now? Nothing else? Please confirm @cammellos @adambabik

Reply:
Suppose*